### ISSUE TITLE
removeDeprecatedMetadata doesn't work with just \n

### ISSUE THREAD

**Issue by ggailey777** [craigshoemaker/tensies#3]
For me the `removeDeprecatedMetadata` rule doesn't work on my Windows machine. 
After I change the definition to the following that uses `\r\n` it does work for me. 
I wonder if I have a different end-of-line setting in my console? 

```js
{
    id: "removeDeprecatedMetadata",
    description: "Remove deprecated metadata (values defined in config)",
    getPattern: config => {
      const { deprecated } = config.metadata;
      const suffix = ":.*\r\n";
      const expression = deprecated.join(`${suffix}|`) + suffix;
      const regex = new RegExp(expression, "g");
      return regex;
    },
    run: function(content, config) {
      return content.replace(this.getPattern(config), "");
    }
  }
```

**Comment by craigshoemaker** [comment]
I tried with `\n\r` and it doesn't work on my machine. I'll have to look into this some more to find out why.

**Comment by ggailey777** [comment]
You must be using Unix-style endings (\n) and I'm using Windows style (\r\n), a description is [here](http://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html). 
This is maybe something we can set in the config?

Then it occurs to me that the line endings are going to vary based on what people are checking in to GitHub, no?

**Comment by craigshoemaker** [comment]
yeah - after giving it some thought, I realized I just need to update the regex to support both 😉

So now the pattern is: `:.*[\n|\r\n]` 

Closed by 4ca3a2ee6c5df57b870e7bca909d3c9dc31742ac
